ds3-cli-certification/src/test/java/com/spectralogic/ds3cli/certification/Certification_Test_7.java

```diff
@@ -0,0 +1,454 @@
+        OUT.close();
+    }
+
+    @Test
+    public void test_7_1_create_bucket() throws Exception {
+        final String testDescription = "7.1: Create Bucket";
+        final String bucketName = CertificationUtil.getBucketName(testDescription);
+        OUT.startNewTest(testDescription);
+        boolean success = false;
+
+        try {
+            final String expectedCreateBucketMsg = "Success: created bucket " + bucketName + ".";
+            OUT.insertLog("Create bucket");
+            final String createBucketCmd = "--http -c put_bucket -b " + bucketName;
+            final CommandResponse createBucketResponse = Util.command(client, createBucketCmd);
+            OUT.insertCommand(createBucketCmd, createBucketResponse.getMessage());
+            assertThat(createBucketResponse.getMessage(), is(expectedCreateBucketMsg));
+
+            OUT.insertLog("List all buckets");
+            final String listBucketCmd = "--http -c get_service";
+            final CommandResponse listBucketResponse = Util.command(client, listBucketCmd);
+            OUT.insertCommand(listBucketCmd, listBucketResponse.getMessage());
+            assertThat(listBucketResponse.getMessage(), containsString(bucketName));
+            success = true;
+        } finally {
+            OUT.finishTest(testDescription, success);
+            Util.deleteBucket(client, bucketName);
+        }
+    }
+
+    @Test
+    public void test_7_2_1_invalid_credentials() throws Exception {
+        final String testDescription = "7.2.1: Invalid Credentials";
+        final String endpoint = System.getenv("DS3_ENDPOINT");
+        final Credentials creds = new Credentials("invalid_access_id", "invalid_secret_key");
+        final Ds3Client invalid_client = Ds3ClientBuilder.create(endpoint, creds).withHttps(false).build();
+        OUT.startNewTest(testDescription);
+        boolean success = false;
+
+        try {
+            Util.command(invalid_client, "--http -c get_service");
+        } catch(final FailedRequestException e) {
+            final String formattedException = FailedRequestExceptionHandler.format(e);
+            OUT.insertPreformat(formattedException);
+            final String expectedError = "permissions / authorization error";
+            assertThat(formattedException, containsString(expectedError));
+            success = true;
+        } finally {
+            OUT.finishTest(testDescription, success);
+        }
+    }
+
+    @Test
+    public void test_7_2_2_invalid_endpoint() throws Exception {
+        final String testDescription = "7.2.2: Invalid Endpoint";
+        final String endpoint = "invalid_endpoint";
+        final Credentials creds = new Credentials("invalid_access_id", "invalid_secret_key");
+        final Ds3Client invalid_client = Ds3ClientBuilder.create(endpoint, creds).build();
+        OUT.startNewTest(testDescription);
+        boolean success = false;
+        try {
+            Util.command(invalid_client, "--http -c get_service");
```

Comment:
This should have a fail() statement after the Util.command, otherwise you'll get a false positive